ts/components/conversation/SessionQuotedMessageComposition.tsx

```diff
@@ -68,6 +70,8 @@ export const SessionQuotedMessageComposition = () => {
 
   const { author, attachments, text: quoteText } = quotedMessageProps || {};
 
+  const isPublic = useIsPublic(quotedMessageProps?.convoId);
```

Comment:
Composition is always going to render the currently selected conversation right?
If yes, you can use `useIsSelectedPublic` instead.
The useIsPublic is less efficient